docs/src/pages/ui/getting-started/installation/index.page.mdx

```diff
@@ -16,3 +16,7 @@ import Link from 'next/link';
 Amplify UI ships with a default [theme](/ui/theme) that you can customize to match the look and feel of your project.
 
 <Fragment>{({ platform }) => import(`./styles.${platform}.mdx`)}</Fragment>
+
+<Fragment platforms={['angular']}>
+  {({ platform }) => import(`./troubleshooting.${platform}.mdx`)}
```

Comment:
Since this is just a single platform, you could explicitly reference the file & save a bit on bundling time.

```suggestion
  {() => import(`./troubleshooting.angular.mdx`)}
```

But honestly, I doubt we'll see much difference.